auth/grpctransport/directpath.go

```diff
@@ -121,3 +128,37 @@ func configureDirectPath(grpcOpts []grpc.DialOption, opts *Options, endpoint str
 	}
 	return grpcOpts, endpoint
 }
+
+// OnComputeEngine returns whether the client is running on GCE.
+//
```

Comment:
Can you explicitly document here how this function works, and how it differs from the `metadata.OnGCE` func that is used elsewhere in this library? If it returns false for Serverless environments, that seems worth noting.

Reply:
thanks, I updated the comments.